build.sc

```diff
@@ -131,7 +131,8 @@ object scalalib extends MillModule {
   def moduleDeps = Seq(main)
 
   def ivyDeps = Agg(
-    ivy"org.scala-sbt:test-interface:1.0"
+    ivy"org.scala-sbt:test-interface:1.0",
+    ivy"org.scala-lang.modules::scala-parser-combinators:1.1.0"
```

Comment:
Parser combinators are used for parsing the dependencies version numbers

Reply:
not fastparse?

Reply:
Yeah if you're going to use a combinator library just use fastparse, after all we already have it as a transitive dependency through Ammonite

Reply:
I took the parsing code straight from from sbt-updates, which uses scala-parser-combinators ; but if we already have fastparse as a dependency, I can certainly try to rewrite that code with it. 👍

Reply:
@lihaoyi turned out to be pretty easy to rewrite with fastparse (as an added bonus, I think it's now a bit more readable than the original code which used regex-based parsers). 👍